File: src/main/java/com/nedap/retail/rheas/core/api/model/Subscription.java
package com.nedap.retail.rheas.core.api.model;

/**
 * Subscription information.
 * 
 * @author Jasper Kremer
 */
public class Subscription {

    public Integer id;
    public String specname;
    public String notification_uri;
    public String extern_ref;
    /**
     * Amount lease time in minutes.
     */
    public int lease;

    public Subscription(final Integer id, final String specname, final String notification_uri,
            final String extern_ref, final int lease) {
        this.id = id;
        this.specname = specname;
        this.notification_uri = notification_uri;
        this.extern_ref = extern_ref;
        this.lease = lease;
    }

    @Override
    public String toString() {
        StringBuilder result = new StringBuilder();
        result.append("id = ");
        result.append(this.id);
        result.append("\n");
        result.append("specname = ");
        result.append(this.specname);
        result.append("\n");
        result.append("notification uri = ");
        result.append(this.notification_uri);
        result.append("\n");
        result.append("external reference = ");
        result.append(this.extern_ref);
        result.append("\n");
        result.append("lease time = ");
        result.append(this.lease);
        result.append(" minutes\n");
        return result.toString();
    }
}

File: csharp/Subscription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace StoreIDTopGateAPI
{
    [DataContract]
    class Subscription
    {
        [DataMember]
        public int id { get; set; }
        [DataMember]
        public String specname { get; set; }
        [DataMember]
        public String notification_uri { get; set; }
        [DataMember]
        public String extern_ref { get; set; }
        [DataMember]
        public int lease { get; set; }

        public Subscription(int id, String specname, String notification_uri, String extern_ref, int lease) {
            this.id = id;
            this.specname = specname;
            this.notification_uri = notification_uri;
            this.extern_ref = extern_ref;
            this.lease = lease;
        }

        public override String ToString() {
            StringBuilder result = new StringBuilder();
            result.Append("id = ");
            result.Append(this.id);
            result.Append("\n");
            result.Append("specname = ");
            result.Append(this.specname);
            result.Append("\n");
            result.Append("notification uri = ");
            result.Append(this.notification_uri);
            result.Append("\n");
            result.Append("external reference = ");
            result.Append(this.extern_ref);
            result.Append("\n");
            result.Append("lease time = ");
            result.Append(this.lease);
            result.Append(" minutes\n");
            return result.ToString();
        }
    }
}
